x-pack/plugins/alerting/server/rules_client/tests/get_action_error_log.test.ts

```diff
@@ -0,0 +1,630 @@
+      ecs: {
+        version: '1.8.0',
+      },
+    },
+    {
+      '@timestamp': '2022-03-23T17:37:07.096Z',
+      event: {
+        provider: 'actions',
+        action: 'execute',
+        kind: 'action',
+        start: '2022-03-23T17:37:07.095Z',
+        end: '2022-03-23T17:37:07.096Z',
+        duration: '1000000',
+        outcome: 'failure',
+      },
+      kibana: {
+        alert: {
+          rule: {
+            execution: {
+              uuid: '08d9b0f5-0b41-47c9-951f-a666b5788ddc',
+            },
+          },
+        },
+        saved_objects: [
+          {
+            rel: 'primary',
+            type: 'action',
+            id: '9e67b8b0-9e2c-11ec-bd64-774ed95c43ef',
+            type_id: '.server-log',
+          },
+          {
+            rel: 'primary',
+            type: 'alert',
+            id: 'a348a740-9e2c-11ec-bd64-774ed95c43ef',
+            type_id: 'example.always-firing',
+          },
+        ],
+        task: {
+          scheduled: '2022-03-23T17:37:04.677Z',
+          schedule_delay: 2418000000,
+        },
+        server_uuid: '5b2de169-2785-441b-ae8c-186a1936b17d',
+        version: '8.2.0',
+      },
+      message: 'action execution failure: .server-log:9e67b8b0-9e2c-11ec-bd64-774ed95c43ef: s',
+      error: {
+        message:
+          'an error occurred while running the action executor: something funky with the server log',
+      },
+      ecs: {
+        version: '1.8.0',
+      },
+    },
+    {
+      '@timestamp': '2022-03-23T17:37:07.086Z',
+      event: {
+        provider: 'actions',
+        action: 'execute',
+        kind: 'action',
+        start: '2022-03-23T17:37:07.084Z',
+        end: '2022-03-23T17:37:07.086Z',
+        duration: '2000000',
+        outcome: 'failure',
+      },
+      kibana: {
+        alert: {
+          rule: {
+            execution: {
+              uuid: '08d9b0f5-0b41-47c9-951f-a666b5788ddc',
+            },
+          },
+        },
+        saved_objects: [
+          {
+            rel: 'primary',
+            type: 'action',
+            id: '9e67b8b0-9e2c-11ec-bd64-774ed95c43ef',
+            type_id: '.server-log',
+          },
+          {
+            rel: 'primary',
+            type: 'alert',
+            id: 'a348a740-9e2c-11ec-bd64-774ed95c43ef',
+            type_id: 'example.always-firing',
+          },
+        ],
+        task: {
+          scheduled: '2022-03-23T17:37:04.678Z',
+          schedule_delay: 2406000000,
+        },
+        server_uuid: '5b2de169-2785-441b-ae8c-186a1936b17d',
+        version: '8.2.0',
+      },
+      message: 'action execution failure: .server-log:9e67b8b0-9e2c-11ec-bd64-774ed95c43ef: s',
+      error: {
+        message:
+          'an error occurred while running the action executor: something funky with the server log',
+      },
+      ecs: {
+        version: '1.8.0',
+      },
+    },
+    {
+      '@timestamp': '2022-03-23T17:23:05.249Z',
+      event: {
+        provider: 'alerting',
```

Comment:
nit: would probably alerting entries from the test since we're no longer querying for them.